recipe/meta.yaml

```diff
@@ -1,23 +1,25 @@
-{% set version = "1.5.4" %}
+{% set version = "1.5.5" %}
 
 package:
   name: pkgconfig
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/pkgconfig/pkgconfig-{{ version }}.tar.gz
-  sha256: c34503829fd226822fd93c902b1cf275516908a023a24be0a02ba687f3a00399
+  sha256: deb4163ef11f75b520d822d9505c1f462761b4309b1bb713d08689759ea8b899
 
 build:
   number: 0
+  # noarch: python
+  # skip s390x due to missing the package poetry for the s390x architecture
+  # skip: True  # [py<36 or s390x]
+  skip: True  # [s390x]
   script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
   host:
     - python
     - pip
-    - nose >=1.0
-    - poetry
```

Comment:
Eliminate nose and poetry since those dependencies are no longer needed:

https://github.com/conda-forge/pkgconfig-feedstock/commit/17c30d27e334b7f37137a9b5a045823f83e93742#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9a

https://github.com/matze/pkgconfig/commit/7280360cd9f5f71c98266a86070fb7477bf73c77#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711

Reply:
The c-f feedstock swapped `poetry` for `poetry_core`, and that is not reflected in this change set.

Reply:
`poetry_core` is in the recipe as `poetry-core` :
 https://github.com/AnacondaRecipes/pkgconfig-feedstock/blob/update_to_1.5.5/recipe/meta.yaml#L19